modules/fbs-core/math-parser/src/main/kotlin/de/thm/ii/fbs/mathParser/SemanticAstComparator.kt

```diff
@@ -1,21 +1,58 @@
 package de.thm.ii.fbs.mathParser
 
 import de.thm.ii.fbs.mathParser.ast.*
-import de.thm.ii.fbs.mathParser.transformer.*
+import de.thm.ii.fbs.mathParser.transformers.*
 import java.math.RoundingMode
 
-class SemanticAstComparator(private val decimals: Int, private val roundingMode: RoundingMode = RoundingMode.HALF_UP) {
+class SemanticAstComparator(
+    decimals: Int = 2,
+    roundingMode: RoundingMode = RoundingMode.HALF_UP,
+    ignoreNeutralElements: Boolean = false,
+    applyInverseElements: Boolean = false,
+    applyCommutativeLaw: Boolean = false,
```

Comment:
These lines have no code coverage, to get full branch coverage you need to initialise the class in a test with no parameters `val semanticAstComparator = SemanticAstComparator()`  (see [codecov](https://app.codecov.io/gh/thm-mni-ii/feedbacksystem/pull/1165#654b61ba08e15d8ac6f49ee6f609e235-R8)).